frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -27,7 +27,6 @@ local MODE = {
     book_time_to_read = 6,
     chapter_time_to_read = 7,
     frontlight = 8,
-    mem_usage = 9,
```

Comment:
Besides memory leak issues I also think it's cool/useful to know how much memory a given document is using. But I'm probably not quite representative of a regular user in that regard. :-P